dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Company.java

```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ */
+package val31.web;
+
+import jakarta.validation.GroupSequence;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.groups.ConvertGroup;
+import jakarta.validation.groups.Default;
+
+/**
+*
+*/
+@GroupSequence({ CompanyChecks.class, Company.class })
```

Comment:
This can be removed, we aren't testing group sequences yet (though we may want to add a records group sequence test after this PR 🤔 )

Reply:
updated